Makefile

```diff
@@ -1,13 +1,41 @@
-all: build
+REGISTRY?=directxman
+IMAGE?=k8s-prometheus-adapter
+TEMP_DIR:=$(shell mktemp -d)
+ARCH?=amd64
+ALL_ARCH=amd64 arm arm64
+ML_PLATFORMS=linux/amd64,linux/arm,linux/arm64
+
+VERSION?=latest
 
+ifeq ($(ARCH),amd64)
+        BASEIMAGE?=busybox
+endif
+ifeq ($(ARCH),arm)
+        BASEIMAGE?=armhf/busybox
+endif
+ifeq ($(ARCH),arm64)
+        BASEIMAGE?=aarch64/busybox
+endif
+
+all: build
 build:
```

Comment:
Actually this builds things for just one arch where amd64 is the default. However, I'll split it to `build` and `docker-build`